bundles/admin/admin-layereditor/view/AdminLayerForm/VisualizationTabPane/VectorStyle/VectorStyleSelect.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { List, Checkbox } from 'antd';
+import { EditOutlined, DeleteOutlined } from '@ant-design/icons';
+import { Button } from 'oskari-ui';
+import { Controller } from 'oskari-ui/util';
+import styled from 'styled-components';
+
+const StyledItem = styled(List.Item)`
+    & {
+        background: #ffffff;
+        line-height: 30px;
+        padding: 0 11px;
+    }
+`;
+
+export const StyleSelect = (props) => {
+    return (
```

Comment:
Check that styles exists.
const styles = props.layer.options.styles;
if (!styles) return null;

Reply:
Checks added!

Reply:
You should check that it's an object since you are using it for Object.keys()

Reply:
See https://github.com/kasparia/oskari-frontend/pull/2